ddmrp/models/stock_buffer.py

```diff
@@ -982,6 +982,7 @@ def _past_moves_domain(self, date_from, date_to, locations):
             ("state", "=", "done"),
             ("location_id", "in", locations.ids),
             ("location_dest_id", "not in", locations.ids),
+            ("location_dest_id.usage", "!=", "inventory"),
```

Comment:
Could you include an extra domain based on a company setting to be able to include scraps, something like:
```python
if not self.env.company.ddmrp_adu_calc_include_scrap:
    domain.append(("location_id.scrap_location", "=" , False))
```

cc @jgrandguillaume I have checked the option to use a production location as a scrap location but Odoo only lets you use internal or inventory loss location to scrap.